src/library/scala/collection/mutable/ListBuffer.scala

```diff
@@ -120,31 +120,35 @@ class ListBuffer[A]
     this
   }
 
-  // Overridden for performance
```

Comment:
as opposed to `// Overperformant for good riddance`.

Reply:
it just seemed really odd to me, as if that's the only method we override anywhere for performance